build_tools/buildkite/pipelines/postsubmit.yml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+agents:
+  queue: "orchestration"
+  security: "submitted"
+
+steps:
+  - label: ":hiking_boot: Bootstrapping postsubmit pipeline"
+    env:
+      PIPELINE_BOOTSTRAPPED: ${PIPELINE_BOOTSTRAPPED:-false}
+    commands: |
+      ./build_tools/buildkite/scripts/bootstrap_pipeline.sh \
+          build_tools/buildkite/pipelines/postsubmit.yml
+
+  - wait
+
+  - label: "Checking if pipelines should be updated"
+    key: "update-pipelines"
+    commands: |
+      export BUILDKITE_ACCESS_TOKEN="$(gcloud secrets versions access latest \
+          --secret=iree-buildkite-privileged)"
+      # Note that dollar sign has to be escaped to avoid Buildkite trying to do
+      # a substitution.
+      ./build_tools/buildkite/scripts/check_for_later_completed_build.py || ret=\$?
```

Comment:
I'm probably not familiar enough with Buildkite. But why don't we put the check into update_pipeline_configuration.py? Just have a feeling that there will be a race condition between running check_for_later_build.py and update_pipelines.yml.

For example, a later build B didn't start when the earlier build A ran check_for_later_build.py, then for some reason the build A was staled and build B finished update_pipelines.yml first. The build A continues again and overwrite the later build B.